core/src/main/kotlin/fr/sncf/osrd/api/pathfinding/RemainingDistanceEstimator.kt

```diff
@@ -13,6 +13,13 @@ import fr.sncf.osrd.utils.units.Offset
 /**
  * This is a function object that estimates the remaining distance to the closest target point,
  * using geo data. It is used as heuristic for A*.
+ *
+ * **LEGACY**
+ *
+ * Not currently in used as the geodesic computations are too expensive to compute, resulting in a
+ * net performance loss.
+ *
+ * TODO: before reusing this class, it now needs to return a minimum *time* instead of a distance.
```

Comment:
I don't really this the value in keeping this class anymore, shouldn't we just delete it?

Reply:
It's supposed to be fixed and used soon-ish. But that's the second comment on this, so I'll just remove it and link the removal in the relevant issue.